test/anim.test.js

```diff
@@ -15,17 +15,17 @@ test('fake timers', () => {
 
   new DMX.Animation().add({
     1: 255,
-  }, 100).add({
```

Comment:
Is this just to speed up the test?

Reply:
Node 20 seems to behave differently with timers internally so we hit a Jest internal timer limit.

Reply:
Node 16 and 18 pass the tests just fine with it set to `100`.